### PR TITLE
getProxyUrl method refactoring

### PR DESCRIPTION
**Pull request by churkin**
\cc @LavrovArtem  @miherlosev

### CODE REVIEW

**Comment by churkin** [comment]
@testcafe-build-bot \retest

File: src/client/utils/url.js [hunk 1]
```diff
@@ -95,7 +110,13 @@ export function changeDestUrlPart (proxyUrl, prop, value, resourceType) {
         var proxy     = parsed.proxy;
         var destUrl   = urlResolver.changeUrlPart(parsed.destUrl, prop, value, document);
 
-        return getProxyUrl(destUrl, proxy.hostname, proxy.port, sessionId, resourceType);
+        return getProxyUrl(destUrl, {
+            proxyHostname: proxy.hostname,
+            proxyPort:     proxy.port,
+
```
Comment (miherlosev) [inline src/client/utils/url.js]: remove empty line

File: src/processing/dom/index.js [hunk 2]
```diff
@@ -456,10 +456,16 @@ export default class DomProcessor {
...
                             var proxyHostname      = urlUtils.parseUrl(location).hostname;
+                            var proxyPort          = this.adapter.getCrossDomainPort();
                             var iframeResourceType = urlUtils.getResourceTypeString({ isIframe: true });
 
-                            proxyUrl = resourceUrl ? this.adapter.getProxyUrl(resourceUrl, proxyHostname,
-                                this.adapter.getCrossDomainPort(), proxyUrlObj.sessionId, iframeResourceType) : '';
+                            proxyUrl = resourceUrl ? this.adapter.getProxyUrl(resourceUrl, {
+                                proxyHostname,
+                                proxyPort,
+
```
Comment (miherlosev) [inline src/processing/dom/index.js]: remove empty line

**Comment by miherlosev** [comment]
\r-. Minor style issues

File: src/client/utils/url.js (hunk 1 above)
Comment (churkin) [inline src/client/utils/url.js]: needed for a nice format

File: src/processing/dom/index.js (hunk 2 above)
Comment (churkin) [inline src/processing/dom/index.js]: needed for a nice format

File: src/client/utils/url.js (hunk 1 above)
Comment (miherlosev) [inline src/client/utils/url.js]: Ok. Problem with autoformatting.

**Comment by miherlosev** [comment]
lgtm

**Comment by LavrovArtem** [comment]
lgtm

**Pull request merged**